src/components/NewCommentForm.tsx

```diff
@@ -14,7 +102,13 @@ export const NewCommentForm: React.FC = () => {
             name="name"
             id="comment-author-name"
             placeholder="Name Surname"
-            className="input is-danger"
+            className={classNames('input', {
+              'is-danger': hasNameError,
+            })}
+            value={comment.name}
+            onChange={e => {
+              handleChangeName(e);
+            }}
```

Comment:
```suggestion
            onChange={handleChangeName}
```
additional (wrapper) function is redundant